ci/driver.sh

```diff
@@ -23,48 +23,62 @@ repo_url=${repo_url:-"https://github.com/NOAA-EMC/global-workflow.git"}
 ################################################################
 # Setup the reletive paths to scripts and PS4 for better logging 
 ################################################################
-set -eux
-pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+script_pwd="$(cd "$(dirname  "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
 scriptname=$(basename "${BASH_SOURCE[0]}")
 echo "Begin ${scriptname} at $(date -u)" || true
 export PS4='+ $(basename ${BASH_SOURCE})[${LINENO}]'
 
-host=$(hostname) || true
-
 usage() {
   set +x
   echo
   echo "Usage: $0 -t <target> -h"
   echo
-  echo "  -t  target/machine script is running on    DEFAULT: ${host}"
-  echo "  -h  display this message and quit"
+  echo    "  -h  display this message and quit"
+  echo -n "  -t  target/machine script is running"
+  if [[ -n "${TARGET+x}" ]]; then
+    echo "on DEFAULT: ${TARGET}"
+  fi  
   echo
   exit 1
 }
 
+
+
 #########################################################################
 #  Set up runtime environment varibles for accounts on supproted machines
 #########################################################################
-TARGET=$(hostname) || true
-export TARGET
-while getopts "t:h" opt; do
-  case ${opt} in
-    t)
-      TARGET=${OPTARG}
-      ;;
-    h|\?|:)
-      usage
-      ;;
-    *)
-      echo "Unrecognized option" 
-      usage
-  esac
-done
+
+source "${script_pwd}/../ush/detect_machine.sh"
```

Comment:
```suggestion
source "${ROOT_DIR}/ush/detect_machine.sh"
```
In light of the comment above.